src/applications/static-pages/authenticated-homepage/tests/fixtures/feature-toggles.json

```diff
@@ -0,0 +1,6 @@
+{
+  "data": {
+    "type": "feature_toggles",
+    "features": [{ "name": "show_authenticated_homepage", "value": true }]
```

Comment:
If we use JS for this, rather than JSON, we could leverage the constant for the feature flag name. 🤔